MediaBrowser.MediaEncoding/Encoder/EncoderValidator.cs

```diff
@@ -44,18 +44,7 @@ public class EncoderValidator
             "mpeg4_cuvid",
             "vp8_cuvid",
             "vp9_cuvid",
-            "av1_cuvid",
-            "h264_mmal",
-            "mpeg2_mmal",
-            "mpeg4_mmal",
-            "vc1_mmal",
-            "h264_opencl",
-            "hevc_opencl",
-            "mpeg2_opencl",
-            "mpeg4_opencl",
-            "vp8_opencl",
-            "vp9_opencl",
-            "vc1_opencl"
```

Comment:
Why remove them too?

Reply:
Actually these is no opencl decoder in ffmpeg. I don't know why they were added.
https://github.com/FFmpeg/FFmpeg/search?q=h264_opencl